IFC4x3/PropertyConstants/c/CONICAL_26RuieLNzFzgzGBCuts7cB.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONICAL_26RuieLNzFzgzGBCuts7cB" Name="CONICAL" UniqueId="866f8b28-557f-4ff6-af50-2cce37d8798b" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)